src/components/card/product-card/ProductCard.jsx

```diff
@@ -0,0 +1,120 @@
+import "./product-card.css";
+import PropTypes from "prop-types";
+import { ROUTE_HOME } from "../../../utils";
+
+const ProductCard = ({ details, loading, cardIsOnPage }) => {
+  const {
+    ratings,
+    isInWishList,
+    img: { altText, url },
+    heading,
+    description,
+    price: { current, earlier, offPct },
+    isInCart,
+  } = details;
+
+  const [wishlistIconType, wishlistTooltipText] = isInWishList
+    ? ["fas", "remove from wishlist"]
```

Comment:
While this is okay but things can get unnecessarily complex if you want some more conditional classnames.
Take a look at this package https://www.npmjs.com/package/classnames
See if it can help you write cleaner code.

Reply:
Yeah, there was only one class name, that's why I didn't explore much.

But thanks, will look at the package you have shared.